### PR TITLE
Added ops metrics

### PR DESCRIPTION
**Pull request by psav**

### CODE REVIEW

**Comment by BlakeHolifield** [comment]
It seems like most of this is also in #182. Are you thinking you'll keep the metric changes in there or split them out here?

File: controllers/cloud.redhat.com/proxy_client.go [hunk 1]
```diff
@@ -21,21 +22,34 @@ type ProxyClient struct {
 	client.Client
 }
 
+func (p *ProxyClient) Get(ctx context.Context, key types.NamespacedName, obj runtime.Object) error {
+	clientOpsMetric.With(prometheus.Labels{"operation": "get"}).Inc()
```
Comment (kylape) [inline controllers/cloud.redhat.com/proxy_client.go:26]: Are you okay knowing that this get will often return locally cached data?

Comment (psav) [reply controllers/cloud.redhat.com/proxy_client.go:26]: Yes

Comment (psav) [reply controllers/cloud.redhat.com/proxy_client.go:26]: It's just to track these things

**Pull request merged**